pkg/virt-operator/creation/components/webhooks.go

```diff
@@ -476,7 +476,11 @@ func NewVirtAPIValidatingWebhookConfiguration(installNamespace string) *v1beta1.
 					Rule: v1beta1.Rule{
 						APIGroups:   []string{virtv1.GroupName},
 						APIVersions: virtv1.ApiSupportedWebhookVersions,
-						Resources:   []string{"*/status"},
+						Resources: []string{
+							"virtualmachines/status",
+							"virtualmachineinstancereplicasets/status",
+							"virtualmachineinstancemigrations/status",
+						},
```

Comment:
So, the problem here is that virt-api is validating the the KubeVirt CR's status, which means when we create this webhook, if virt-api doesn't come online for some reason we actually can't write to the KubeVirt CR anymore. Good catch here, that's not a great scenario.

Should we still have KubeVirt status validation occuring though? and just move that validation to virt-operator similar to how the KubeVirt spec is validated?

Reply:
From my understanding the only thing the validation webhook was validating for the KubeVirt CR status on the webhook is the schema https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/webhooks/validating-webhook/admitters/status-admitter.go#L16-L18 which I believe should be done via the kubernetes api since we have the openapi validation specified on the CRD.  Is it redundant to also validate it with a validation webhook?

Reply:
In general we want our own vaildation too (more strict), but fur a status update on the KubeVirt CR, I don't think we need it right now. This was really just a not-thought-of side-effect on my side.

Reply:
alrighty, works for me then